src/components/Dashboard/Tiles/Widgets/WidgetsRemoveModal.tsx

```diff
@@ -7,14 +7,15 @@ import { useConfigStore } from '../../../../config/store';
 
 export type WidgetsRemoveModalInnerProps = {
   widgetId: string;
+  widgetType: string;
 };
 
 export const WidgetsRemoveModal = ({
   context,
   id,
   innerProps,
 }: ContextModalProps<WidgetsRemoveModalInnerProps>) => {
-  const { t } = useTranslation([`modules/${innerProps.widgetId}`, 'common']);
+  const { t } = useTranslation([`modules/${innerProps.widgetType}`, 'common']);
```

Comment:
Did you remember to change namespaces for the translation files?

Reply:
This would work because you just renamed Id to Type right ? I don’t see an issue